dataactcore/models/jobTrackerInterface.py

```diff
@@ -193,9 +192,9 @@ def checkJobDependencies(self,jobId):
         for depJobId in self.getDependentJobs(jobId):
             isReady = True
             if not (self.getJobStatus(depJobId) == self.getJobStatusId('waiting')):
-                CloudLogger.logError("Job dependency is not in a 'waiting' state",
-                                     ResponseException("Job dependency is not in a 'waiting' state",StatusCode.CLIENT_ERROR, ValueError),
-                                     traceback.extract_stack())
+                _exception_logger.error(
```

Comment:
Somewhat related -- I noticed that a lot of these log messages don't contain too much context. They're just "The state is wrong, line 55". I'm _hoping_ that we can set user and submission ids on `g` and then retrieve them in the logger to tack that information on, if it's available.